plugins/rapid7/mask.yaml

```
@@ -0,0 +1,96 @@
+connection_type:
+  name: rapid7
+  attributes:
+    REQUEST_URL:
+      description: Rapid7 API Adress
+      display_name: API Address
+      field_type: 0
+      index: 1
+      input_type: text
+      placeholder: Rapid7 API URL
+      required: true
+    USERNAME:
+      description: Rapid7 Username
+      display_name: Username
+      field_type: 0
+      index: 2
+      input_type: text
+      placeholder: Rapid7 Username
+      required: true
+    PASSWORD:
+      description: Rapid7 Password
+      display_name: Password
+      field_type: 0
+      index: 3
+      input_type: password
+      placeholder: Rapid7 Password
+      required: true
+collection:
+  name: rapid7
+  icon_uri: "https://app.dev.blinkops.com/assets/images/Icons/Companies/Company/XL/Rapid7.svg"
+  active: true
+is_connection_optional: false
+request_url_conn_type: rapid7
```

```suggestion
request_url_conn_type: rapid7
auto_paging_params:
  - page
  - size
```
You need this if you want the generator to generate the auto paging param